e2e/_suites/fleet/features/fleet_mode_agent.feature

```diff
@@ -7,58 +7,56 @@ Background: Setting up kibana instance with the default profile
 
 @install
 @skip:windows
-Scenario Outline: Deploying the agent
+Scenario: Deploying the <os> agent
   Given an agent is deployed to Fleet with "tar" installer
   When the "elastic-agent" process is in the "started" state on the host
   Then the agent is listed in Fleet as "online"
     And system package dashboards are listed in Fleet
 
-# @enroll
-# @skip:windows
-# Scenario Outline: Deploying the agent with enroll and then run on rpm and deb
-#   Given an agent is deployed to Fleet
-#   When the "elastic-agent" process is in the "started" state on the host
-#   Then the agent is listed in Fleet as "online"
-#     And system package dashboards are listed in Fleet
-
-# @centos
-# Examples: Centos
-# | os     |
-# | centos |
-
-# @debian
-# Examples: Debian
-# | os     |
-# | debian |
+@enroll
+@skip:windows
+Scenario: Deploying the <os> agent with enroll and then run on rpm and deb
```

Comment:
```suggestion
Scenario: Deploying the agent with enroll and then run on rpm and deb
```

Reply:
And actually we may want to comment this out for now, at least until we get the step definitions updated to be more agnostic with what type of installer to use